src/xngin/apiserver/routers/experiments/experiments_common.py

```diff
@@ -671,6 +671,12 @@ async def get_existing_assignment_for_participant(
             observed_at=existing_assignment.observed_at if hasattr(existing_assignment, "observed_at") else None,
             outcome=existing_assignment.outcome if hasattr(existing_assignment, "outcome") else None,
             context_values=existing_assignment.context_vals if hasattr(existing_assignment, "context_vals") else None,
+            current_mu=existing_assignment.current_mu if hasattr(existing_assignment, "current_mu") else None,
+            current_covariance=existing_assignment.current_covariance
+            if hasattr(existing_assignment, "current_covariance")
+            else None,
+            current_alpha=existing_assignment.current_alpha if hasattr(existing_assignment, "current_alpha") else None,
+            current_beta=existing_assignment.current_beta if hasattr(existing_assignment, "current_beta") else None,
```

Comment:
If you're going to return this info here, we should also inside `create_assignment_for_participant`. Oh, you're also not selecting these extra values from the query above, so `existing_assignment` will never have those values.